crates/cdfy-server/src/room_api.rs

```diff
@@ -99,22 +108,49 @@ async fn message_plugin(
     );
     let store = RedisRoomStore::default();
 
-    let mut room = store.get(room_id).map_err(into_resp)?;
+    let mut room = store.get(room_id.clone()).map_err(into_resp)?;
     room.message(body.user_id.clone(), plugin_id, body.message.clone())
         .map_err(into_resp)?;
+
+    if let Err(e) = notifier.tx.send(Notification::new(

```

Comment:
need to notify other APIs(`join_room()`, `load_plugin`)

Reply:
fixed it at [this commit](https://github.com/wakame-tech/cdfy/pull/4/commits/d9cd1c175efc9c89ea8685010eff26e67a62c104)